utils/createDatabaseConnection.ts

```diff
@@ -0,0 +1,5 @@
+import pgp from 'pg-promise';
+import { appConfig } from '@/utils/appConfig';
+
+export const createDatabaseConnection = () =>
+  pgp()(appConfig.databaseConnectionString);
```

Comment:
Pretty basic right now, idea is we'll call this where we instantiate the apollo server and pass it through to our data sources to our resolvers